### PR TITLE
Add searchGalaxy icon

### PR DESCRIPTION
**Pull request by oliviacpu**
We're doing some CSS tricks to switch out an SVG background and if this were in the icon set we could change props and do things in a more React-y way

### CODE REVIEW

File: lib/icon.js [hunk 1]
```diff
@@ -145,6 +150,33 @@ const svgs = {
       />
     </SVGBase>
   ),
+  searchGalaxy: (props) => (
```
Comment (whimsicallyson) [inline lib/icon.js:153]: is this duplicated in `lib/icon/searchGalaxy.js` now?

File: lib/icon/index.js [hunk 2]
```diff
@@ -0,0 +1,395 @@
+import React from 'react';
```
Comment (whimsicallyson) [inline lib/icon/index.js:1]: how is this different from what's in `lib/icon.js`?

Comment (whimsicallyson) [reply lib/icon/index.js:1]: that's more like it 😁

**Comment by keithk** [comment]
:rocket: PR was released in [`v0.19.0`](https://github.com/glitchdotcom/shared-components/releases/tag/v0.19.0) :rocket:

**Pull request merged**